components/User/AdminUserProfilePage.vue

```diff
@@ -255,7 +255,7 @@
 
         <template #button>
           <ModalWithButton
-            :title="$t('Are you sure you want to delete this organization ?')"
+            :title="$t('Are you sure you want to delete this user ?')"
```

Comment:
:fr: ?

Reply:
I didn't want to mix language :sweat_smile: